src/cfnlint/data/ExtendedSpecs/all/03_value_types.json

```diff
@@ -4,6 +4,33 @@
     "path": "/ValueTypes",
     "value": {
       "AvailabilityZone": {
+        "AllowedValues": [
```

Comment:
We should remove this list here as part of this move:
https://github.com/aws-cloudformation/cfn-python-lint/blob/4eccd3faa7775c16521e2b5b827b7e8c1f9bffd4/src/cfnlint/helpers.py#L82